packages/mockotlpserver/lib/cli.js

```diff
@@ -121,9 +123,20 @@ async function main() {
         process.exit(0);
     }
 
+    
+    const hasWebPrinter = opts.o.some((printerName) => printerName.endsWith('web'));
+    console.log('hasWebPrinter', hasWebPrinter)
+    console.log(opts.o)
+
     const otlpServer = new MockOtlpServer({
         log,
-        services: ['http', 'grpc', 'ui'],
+        // TODO: this is a shortcut to enable/disable the UI printer but
```

Comment:
note for reviewer: when doing this change I thought the dev may want to see traces from previous tests without being forced activate a printer. As for now they're coupled but they don't need to.